x-pack/test/apm_api_integration/tests/entities/logs/service_logs_error_rate_timeseries.spec.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { log, timerange } from '@kbn/apm-synthtrace-client';
+import { first, last } from 'lodash';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+  const apmApiClient = getService('apmApiClient');
+  const apmSynthtraceEsClient = getService('apmSynthtraceEsClient');
+  const logSynthtrace = getService('logSynthtraceEsClient');
+
+  const serviceName = 'synth-go';
+  const start = new Date('2024-01-01T00:00:00.000Z').getTime();
+  const end = new Date('2024-01-01T00:15:00.000Z').getTime() - 1;
+
+  const hostName = 'synth-host';
+
+  type LogsErrorRateTimeseriesReturnType =
```

Comment:
This type is not used in this file, do you know if we need it here? (I also see an exported interface with the same name 🤔 )